mui/utils.py

```diff
@@ -112,3 +115,58 @@ def print_timestamp(*args, **kw):
     """Print with timestamp prefixed (local timezone)"""
     timestamp = datetime.now().astimezone()
     print(f"[{timestamp}]", *args, **kw)
+
+
+def get_function_models() -> typing.List[typing.Tuple[str, Function]]:
```

Comment:
I think we should have a Python [dataclass](https://docs.python.org/3.7/library/dataclasses.html) for the tuple here and name the two fields to make it easier to follow